roachtest: pgx failed

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2045486&tab=buildLog) on [master@c627e3490d30e8ba88f6c7136717a392a054da4e](https://github.com/cockroachdb/cockroach/commits/c627e3490d30e8ba88f6c7136717a392a054da4e):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgx/run_1
	orm_helpers.go:214,orm_helpers.go:144,pgx.go:113,pgx.go:125,test_runner.go:757: 
		Tests run on Cockroach v20.2.0-alpha.1-706-gc627e3490d
		Tests run against pgx v4.6.0
		177 Total Tests Run
		118 tests passed
		59 tests failed
		3 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestTxCommitWhenDeferredConstraintFailure - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blocklist (pgxBlocklist20_2) is available in the artifacts' pgx log

```

<details><summary>More</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=2045486&tab=artifacts#/pgx)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgx.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2046708&tab=buildLog) on [master@17c8048e80935f8a01477416980d18bf39cba1bb](https://github.com/cockroachdb/cockroach/commits/17c8048e80935f8a01477416980d18bf39cba1bb):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgx/run_1
	orm_helpers.go:214,orm_helpers.go:144,pgx.go:113,pgx.go:125,test_runner.go:757: 
		Tests run on Cockroach v20.2.0-alpha.1-708-g17c8048e80
		Tests run against pgx v4.6.0
		177 Total Tests Run
		118 tests passed
		59 tests failed
		3 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestTxCommitWhenDeferredConstraintFailure - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blocklist (pgxBlocklist20_2) is available in the artifacts' pgx log

```

<details><summary>More</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=2046708&tab=artifacts#/pgx)
Related:
- #50751 roachtest: pgx failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgx.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2054002&tab=buildLog) on [master@3a03f3843a8cdf04f82c52753c61cf01b0d2ddcd](https://github.com/cockroachdb/cockroach/commits/3a03f3843a8cdf04f82c52753c61cf01b0d2ddcd):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgx/run_1
	orm_helpers.go:214,orm_helpers.go:144,pgx.go:113,pgx.go:125,test_runner.go:757: 
		Tests run on Cockroach v20.2.0-alpha.1-804-g3a03f3843a
		Tests run against pgx v4.6.0
		177 Total Tests Run
		118 tests passed
		59 tests failed
		3 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestTxCommitWhenDeferredConstraintFailure - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blocklist (pgxBlocklist20_2) is available in the artifacts' pgx log

```

<details><summary>More</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=2054002&tab=artifacts#/pgx)
Related:
- #50813 roachtest: pgx failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50798 roachtest: pgx failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50751 roachtest: pgx failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgx.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgx failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2062841&tab=buildLog) on [master@456a07cfc1e53b87abc7709052e54efb1450e758](https://github.com/cockroachdb/cockroach/commits/456a07cfc1e53b87abc7709052e54efb1450e758):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgx/run_1
	orm_helpers.go:214,orm_helpers.go:144,pgx.go:113,pgx.go:125,test_runner.go:757: 
		Tests run on Cockroach v20.2.0-alpha.1-911-g456a07cfc1
		Tests run against pgx v4.6.0
		177 Total Tests Run
		118 tests passed
		59 tests failed
		3 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v4.TestTxCommitWhenDeferredConstraintFailure - unknown (unexpected)
		For a full summary look at the pgx artifacts 
		
		An updated blocklist (pgxBlocklist20_2) is available in the artifacts' pgx log

```

<details><summary>More</summary><p>


Artifacts: [/pgx](https://teamcity.cockroachdb.com/viewLog.html?buildId=2062841&tab=artifacts#/pgx)
Related:
- #50813 roachtest: pgx failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50798 roachtest: pgx failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50751 roachtest: pgx failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgx.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/pgx.go
+++ b/pkg/cmd/roachtest/pgx.go
@@ -49,7 +49,7 @@ func registerPgx(r *testRegistry) {
 
 		t.Status("installing pgx")
 		if err := repeatRunE(
-			ctx, c, node, "install pgx", "go get -u github.com/jackc/pgx",
+			ctx, c, node, "install pgx", fmt.Sprintf("go get -u github.com/jackc/pgx@%s", supportedTag),
 		); err != nil {
 			t.Fatal(err)
 		}

